efi/generate_binary.py

```diff
@@ -73,8 +73,19 @@ def _add_nx_pefile(args):
     try:
         import pefile
     except ImportError:
-        print("Unable to add NX support to binaries without genpeimg or python3-pefile")
-        sys.exit(1)
+        print("Unable to add NX support to binaries without genpeimg or python3-pefile, adding NX support manually")
+        import os
+        with open(args.outfile, 'r+b') as fh:
```

Comment:
for my tiny brain, what's `r+b`? I did expect to see a `rwb` or something.

Reply:
```r+b``` is for read and write in binary mode. ```+``` allows writing as well as far as I know.

Reply:
TIL, thanks!